NativeAuthSampleApp/ProtectedAPIViewController.swift

```diff
@@ -26,24 +26,29 @@ import MSAL
 import UIKit
 
 class ProtectedAPIViewController: UIViewController {
-    let protectedAPIUrl = "Enter_the_Protected_API_Full_URL_Here"
-    let protectedAPIScopes = ["Enter_the_Protected_API_Scopes_Here"]
+
+    let urlApi1: String? = "Enter_the_Protected_API_Full_URL_Here"
```

Comment:
I know this controller is called protected API View controller so should we name these `protectedAPIUrl1`, `protectedAPIScopes1` etc?

Reply:
They should be capitalised regardless

Reply:
Done.